packages/flutter/lib/src/cupertino/tableview.dart

```diff
@@ -0,0 +1,540 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/src/cupertino/colors.dart';
+import 'package:flutter/widgets.dart';
+import 'package:meta/meta.dart';
+
+@visibleForTesting
+const double kDefaultTableViewCellHeight = 44.0;
+
+@visibleForTesting
+const double kDefaultCellInset = 16.0;
+
+@visibleForTesting
+const double kDefaultDividerHeight = 1.0;
+
+/// WARNING: CupertinoTableView and its associated classes are EXPERIMENTAL. IF
+/// YOU USE THESE APIS, WE WILL BREAK YOU!
+///
+/// An iOS-styled table view.
```

Comment:
Put this on top. A one line version of this doc will appear in class lists like https://docs.flutter.io/flutter/cupertino/cupertino-library.html. In this case, it'll just say WARNING:...

Reply:
I moved it down a line instead of moving it up.  I want to ensure that the warning appears in any generated docs until we're stable.